requirements/ci.txt

```diff
@@ -12,7 +12,7 @@ chardet==3.0.4            # via -r requirements/base.txt, requests
 click-log==0.3.2          # via -r requirements/base.txt, edx-lint
```

Comment:
Doesn't the `upgrade` target in `Makefile` need to change to accommodate the rename?

Reply:
it did need an update in `Makefile`